selfapi.js

```diff
@@ -6,6 +6,11 @@ var https = require('https');
 var nodepath = require('path');
 var url = require('url');
 
+var options = {
```

Comment:
Actually, after re-reading `selfapi.js`, I think you should add `selfapi.options = options` instead, given that we replace `module.exports` at the [bottom of the file](https://github.com/JanitorTechnology/selfapi/blob/6cad7814e2bbfd4f892e25dc40f3237ca5b2a6da/selfapi.js#L630-L631).